### ISSUE TITLE
Build fails because https://arp242.net/termtext is 404

### ISSUE THREAD

**Issue by ismail** [arp242/uni#31]
Currently trying to build results in:

```
; go build
go: arp242.net/termtext@v0.0.0-20210624234749-36b745617c68: unrecognized import path "arp242.net/termtext": 
reading https://arp242.net/termtext?go-get=1: 404 Not Found
```

Also this is a good time to thank _you_ for this very useful tool!

**Comment by arp242** [comment]
Ehm, yeah; I moved most Go import paths to zgo.at. I need to update this one too. In the meanwhile, you can either modify the go.mod file manually or use the binaries on the releases page.

**Comment by ismail** [comment]
Hopefully this is not some stupid error on my side, but did the change:

```
-       arp242.net/termtext v0.0.0-20210624234749-36b745617c68                                                                
+       zgo.at/termtext v0.0.0-20210624234749-36b745617c68  
```

now getting a different error:

```
go: zgo.at/termtext@v0.0.0-20210624234749-36b745617c68: parsing go.mod:
        module declares its path as: arp242.net/termtext
                but was required as: zgo.at/termtext
```

**Comment by arp242** [comment]
Ehm, right; you need to update the version too, and then the import paths; the easiest is to just add a `replace` line for now (and undo the previous change):

    replace arp242.net/termtext => zgo.at/termtext v0.0.0-20211028051427-f6235ebe8320

I want to move this package to zgo.at too; I got some WIP stuff in my local directory for that. I'll finish that up after I'm feeling a bit better and then it should be fixed.

**Comment by ismail** [comment]
> replace arp242.net/termtext => zgo.at/termtext v0.0.0-20211028051427-f6235ebe8320

That worked, thanks a lot!

**Comment by arp242** [comment]
Cheers, sorry for the hassle 😅

**Comment by ismail** [comment]
> Cheers, sorry for the hassle sweat_smile

Was a good learning opportunity :)

**Comment by arp242** [comment]
I fixed it up today and released v2.4.0
